cmd/kubeadm/app/cmd/selfhosting/selfhosting.go

```diff
@@ -58,23 +57,19 @@ const (
 // 9. Do that for the kube-apiserver, kube-controller-manager and kube-scheduler in a loop
 func CreateSelfHostedControlPlane(manifestsDir, kubeConfigDir string, cfg *kubeadmapi.InitConfiguration, client clientset.Interface, waiter apiclient.Waiter, dryRun bool) error {
 	glog.V(1).Infoln("creating self hosted control plane")
-	// Adjust the timeout slightly to something self-hosting specific
+	// Adjust the timeout slightly to something selfhosting specific
 	waiter.SetTimeout(selfHostingWaitTimeout)
 
 	// Here the map of different mutators to use for the control plane's PodSpec is stored
 	glog.V(1).Infoln("getting mutators")
 	mutators := GetMutatorsFromFeatureGates(cfg.FeatureGates)
 
-	// Some extra work to be done if we should store the control plane certificates in Secrets
-	if features.Enabled(cfg.FeatureGates, features.StoreCertsInSecrets) {
-
-		// Upload the certificates and kubeconfig files from disk to the cluster as Secrets
-		if err := uploadTLSSecrets(client, cfg.CertificatesDir); err != nil {
-			return err
-		}
-		if err := uploadKubeConfigSecrets(client, kubeConfigDir); err != nil {
-			return err
-		}
+	// Upload the certificates and kubeconfig files from disk to the cluster as Secrets
```

Comment:
agreed. but this PR does not cover the new command line feature and adding the flags it requires.  As of this PR, this is dead code that cant be run. Once my next PR is in this will be a thing and should be added in as a flag then. I think adding this in this PR is scope creep of the second PR to add the command to pivot the cluster.

Reply:
@Klaven ok, with this but please address this comment in the next PR

Reply:
would you like me to put a `TODO` in the code and commit it to ensure it gets updated in the next PR? or just leave it as is and link these comments when I add the  flags?